microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/core/ProjectLabelManager.java

```diff
@@ -73,7 +73,7 @@ private ProjectLabelManager() {
 	private ProjectLabelInfoEntry getProjectLabelInfo(IProject project, List<String> types) {
 		String uri = JDTMicroProfileUtils.getProjectURI(project);
 		if (uri != null) {
-			return new ProjectLabelInfoEntry(uri, getProjectLabels(project, types));
+			return new ProjectLabelInfoEntry(uri, project.getName(), getProjectLabels(project, types));
```

Comment:
`project.getName()` returns the last segment of the project folder path:
https://help.eclipse.org/2020-06/nftopic/org.eclipse.platform.doc.isv/reference/api/org/eclipse/core/resources/IResource.html#getName()
which is the folder name.

I think we want `project.getDescription().getName()` instead.

Reply:
Thank you!

Reply:
In Eclipse IDE context, it uses the project.getName() to display the project and not the project name coming from the .project:

![image](https://user-images.githubusercontent.com/1932211/86230154-ffd42780-bb90-11ea-8cc4-14eff8072d7d.png)

I think we should follow this idea. @fbricon what do you think about that?

Reply:
As of `1a60ddb` I am doing:

```java
project.getName()
```

However, if all this does is get the folder name, then this PR just adds a bunch of overhead to the process, since `vscode-quarkus` was already just using the folder name.